nix/materialized.paymentserver/default.nix

```diff
@@ -2,8 +2,11 @@
   extras = hackage:
     {
       packages = {
+        "network" = (((hackage.network)."3.1.2.7").revisions).default;
+        "stripe-signature" = (((hackage.stripe-signature)."1.0.0.14").revisions).default;
         PaymentServer = ./PaymentServer.nix;
         servant-prometheus = ./.stack-to-nix.cache.0;
+        stripe-core = ./.stack-to-nix.cache.1;
```

Comment:
Everything in `materialized.paymentserver` is generated by Haskell.nix, correct.  Effectively everything in this directory is a cache - it could all be re-generated from `nix/default.nix`.